bayesian/api/api_v2.py

```diff
@@ -76,12 +77,11 @@ def component_vulnerability_analysis_post():
 
     try:
         # Step1: Gather and clean Request
-        packages_list, normalised_input_pkgs = ca_validate_input(input_json, ecosystem)
+        packages_list = ca_validate_input(input_json, ecosystem)
         # Step2: Get aggregated CA data from Query GraphDB,
         graph_response = get_vulnerability_data(ecosystem, packages_list)
         # Step3: Build Unknown packages and Generates Stack Recommendation.
-        stack_recommendation, unknown_pkgs = get_known_unknown_pkgs(
-            ecosystem, graph_response, normalised_input_pkgs)
+        stack_recommendation = get_known_clair_pkgs(ecosystem, graph_response)
```

Comment:
i suggest you keep the function name generic. we can use same vuln for non clair situations too